### ISSUE TITLE
Unable to generate IDEA Projet with sbt gen-idea

### ISSUE THREAD

**Issue by stevben** [killrweather/killrweather#13]
Cannot resolve dependencies with 'sbt gen-idea' (404 errors & long timeouts for each missing dependency). 

![capture d ecran 2015-03-16 a 21 21 41](https://cloud.githubusercontent.com/assets/192763/6675655/7d6d881c-cc22-11e4-87ad-c3175039bff6.png)

**Comment by fedesilva** [comment]
Those timeouts (the visible ones) are from unpublished source and javadoc packages; I get several of those but they do not make gen-idea fail.

Maybe a little bit more info? Also, with the latest IDEA+scala plugin, using sbt-idea is not recommended. IDEA itself should be able to open the folder as a project with full knowledge of sbt files and dependencies.

**Comment by stevben** [comment]
You're right, the topic of this issue is erroneous.  In reality, I do not know if this fails or not because I did not wait for the end of the script because it was too long due to all timeouts (I have only one life :)). 

For IDEA, I already tried but it fails at resolving symbols. It would be fine to describe the configure procedure in README.md.

![capture d ecran 2015-03-16 a 22 26 38](https://cloud.githubusercontent.com/assets/192763/6676891/9999a288-cc2b-11e4-98cc-06c41884edea.png)
